orca-pipelinetemplate/src/test/resources/gate.yml

```diff
@@ -0,0 +1,150 @@
+#document netflix-conventions
+clusters:
+  netflix-conventions:
+    requires:
+      membership:
+        - nf-datacenter
+        - nf-infrastructure
+
+accounts:
+  awsdefault:
+    cloudProvider: aws
+    regions:
+      us-east-1:
+        name: us-east-1 #baerf
+        zones:
+          - us-east-1c
+          - us-east-1d
+          - us-east-1e
+      us-west-2:
+        name: us-west-2  #backrefs?
```

Comment:
name vs the containing key thing kind of sucks - the json backref or the stuff @robzienert was looking at (using a Map type that passes id down to the child) or something else could work here. alternatively just sorting it out in the translation from Map -> typed object could work too